http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +73,36 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("Undertow servlet test HTTP/2")
+    public void undertowHttp2Servlet(Vertx vertx, VertxTestContext vertxTestContext) {
+        URILike baseUri = app.getURI();
+        Checkpoint requestCheckpoint = vertxTestContext.checkpoint(2);
+        HttpClientOptions options = new HttpClientOptions().setProtocolVersion(HttpVersion.HTTP_2);
+        options.setSsl(true)
+                .setUseAlpn(true)
+                .setTrustOptions(new JksOptions().setPath("keystore.jks").setPassword("password"))
+                .setProtocolVersion(HttpVersion.HTTP_2);
+        httpClient = vertx.createHttpClient(options);
+        vertxTestContext
+                .verify(() -> {
+                    httpClient
+                            .request(HttpMethod.GET, app.getURI(Protocol.HTTPS).getPort(), baseUri.getHost(),
```

Comment:
`app.getURI(Protocol.HTTPS).getPort(), baseUri.getHost()` seems bit strange, to make var for host and not for port

Reply:
I've removed that variable to use directly `app.getURI().getHost()`